tests/test_model.py

```diff
@@ -200,3 +201,45 @@ def test_mdu_from_scratch():
     model = FMModel()
     model.filepath = output_fn
     model.save()
+
+
+def test_boundary_with_forcingfile_returns_forcing():
```

Comment:
What is the purpose of having three boundaries on the same test? They are (virtually) doing the same. I believe with only one boundary should suffice.